apps/total-typescript/src/path-to-purchase-react/pricing.tsx

```diff
@@ -380,34 +380,38 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
               <strong>Workshops</strong>
               {modules && (
                 <ul data-workshops="" role="list">
-                  {modules.map((module) => {
-                    const getLabelForState = (state: any) => {
-                      switch (state) {
-                        case 'draft':
-                          return 'Coming soon'
-                        default:
-                          return ''
+                  {modules
+                    .filter((module) => module.moduleType === 'workshop')
```

Comment:
just using a quick filter here so we can separate from `playlist` type when that is added soon. It will appear in "bonus" section.